lib/cbus_elixir/app/app.ex

```diff
@@ -20,9 +19,11 @@ defmodule CbusElixir.App do
 
   """
   def list_speakers do
-    query = from s in Speaker,
+    query = from(s in Speaker,
             join: m in assoc(s, :meeting),
             order_by: [desc: m.date]
```

Comment:
I know this isn't in scope of this PR, but it'd be really cool if you can move the `Repo.preload([:meeting])` on L29 to inside this query

Reply:
Which query do you think it should be moved to?